contracts/structs/JBProjectMetadataData.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+struct JBProjectMetadataData {
```

Comment:
nit) why not just JBProjectMetadata? Similar to JBFundingCycleMetadata

Reply:
yay